### PR TITLE
feat: fb feed for social widget

### PR DESCRIPTION
**Pull request by shsteimer**
Added support for Facebook in social feed for CJ Cup.

See original at: https://www.pgatour.com/tournaments/the-cj-cup.html

Test URLs:
- Before: https://main--theplayers--hlxsites.hlx.page/social
- After: https://social-fb-support--theplayers--hlxsites.hlx.page/social
- CJ Cup FB Version: https://social-fb--pga-the-cj-cup--hlxsites.hlx.page/

### CODE REVIEW

**Comment by fkakatie** [comment]
something in here broke the @source links: 
image

those now point to /undefined in the players and the cj cup drafts examples.

File: blocks/social/social.js [hunk 1]
```diff
@@ -52,42 +53,87 @@ function writeDate(timestamp) {
   return `${day} ${month}`;
 }
 
-function buildTwitterTile(tile, data) {
-  const { user } = data;
-  const profileURL = `${TWITTER_URL}${user.screen_name}`;
+function buildSocialTile(tile, data, network) {
+  const profileURL = data.user.profileUrl;
```
Comment (fkakatie) [inline blocks/social/social.js:57]: my first instinct is that this is where that /undefined is coming from. based on the different formats that this feed provides instagram vs. twitter data, i'm guessing there's probably _not_ one unified path to the profile url in both twitter and facebook data, but i'd love to be wrong.

Comment (shsteimer) [reply blocks/social/social.js:57]: good catch, will work on it a bit.

Comment (shsteimer) [reply blocks/social/social.js:57]: was a simple fix, just has the case of the parameter wrong when I passed it into the function: `profileURL` vs. `profileUrl`

**Review by fkakatie — APPROVED** [review]

File: blocks/social/social.js [hunk 2]
```diff
@@ -1,7 +1,13 @@
-import { readBlockConfig, decorateIcons, fetchPlaceholders } from '../../scripts/scripts.js';
+import {
+  readBlockConfig,
+  decorateIcons,
+  fetchPlaceholders,
+  updateExternalLinks,
```
Comment (fkakatie) [inline blocks/social/social.js:5]: i (finally) noticed these links weren't opening in a new tab, so i snuck in that missing piece...

**Pull request merged**